JoodHajeer/assignment4/Source.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <stdio.h>  
+#include <string>
+#include <queue>
+#include <set>
+#include <stdlib.h>
+#include"codeu_test_lib.h"
+using namespace std;
+
+void DFS(int curRow, int curCol, int numberOfRows, int numberOfColumns, vector<vector<bool> > & tilesMap) {
```

Comment:
Minor thing: pointers are a bit better than references for things you will modify, since it makes it clear to the client.
If you see:
`DFS(..., tilesMap)`
you don't know what will happen with tilesMap, but if you see:
`DFS(..., &tilesMap)`
is clear that tilesMap may be modified.